manifests/p/PostgREST/postgrest/12.2.3/PostgREST.postgrest.locale.en-US.yaml

```diff
@@ -0,0 +1,39 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: PostgREST.postgrest
+PackageVersion: 12.2.3
+PackageLocale: en-US
+Publisher: PostgREST
+PublisherUrl: https://github.com/PostgREST
+PublisherSupportUrl: https://github.com/PostgREST/postgrest/issues
+PackageName: postgrest
+PackageUrl: https://github.com/PostgREST/postgrest
+License: MIT
+LicenseUrl: https://github.com/PostgREST/postgrest/blob/main/LICENSE
+Copyright: |-
+  Copyright (c) 2014 Joe Nelson
+  Copyright (c) 2019 Steve Chavez
+CopyrightUrl: https://github.com/PostgREST/postgrest/blob/main/LICENSE
+ShortDescription: REST API for any Postgres database
+Tags:
+- api
+- automatic-api
+- database
+- haskell
+- http
+- pg
+- pgsql
+- postgres
+- postgresql
+- postgrest
+- rest
+- server
+- sql
+InstallationNotes: |-
+  "libpq.dll" is required by this package. It can be found in any version of PostgreSQL ODBC drivers.
+  Available via "PostgreSQL.psqlODBC" in WinGet. You should manually add it's "bin/" directory (
+  "$env:PROGRAMFILES/psqlODBC/<version>/bin" by default) into "$env:PATH" environment variable.
```

Comment:
So I note here and asking users to resolve "`libpq.dll` not found" error on their own.